spec/models/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Bianca', photo: 'photo.com.uy', bio: 'developer from Uruguay') }
+
+  before { subject.save }
+
+  it 'Name must not be blank.' do
+    expect(subject).to be_valid
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] In accordance to the project requirement, kindly implement test for the user's model's method highlighted from your codebase. 👇 

```
 def recents(limit = 3)
    posts.order(created_at: :desc).limit(limit)
  end
```

<img width="419" alt="Untitled" src="https://github.com/BiancaMe/my-sheeps-blog/assets/103386183/e7a02342-d36a-4389-8c74-84a89a7cad1a">